numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -1374,7 +1374,7 @@ def add_newdoc(place, name, doc):
 
     See Also
     --------
-    ceil, trunc, rint
+    ceil, trunc, rint, fix
 
     Notes
     -----
```

Comment:
Could you also add a mention of `fix` to the note below?

Reply:
Done.